src/php/Analyzer/AnalyzeLiteral.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel\Analyzer;
+
+use Phel\Ast\LiteralNode;
+use Phel\Lang\Phel;
+use Phel\NodeEnvironment;
+
+final class AnalyzeLiteral
+{
+    public function __invoke($x, NodeEnvironment $env): LiteralNode
```

Comment:
Maybe we can add a DocComment for `$x` (`Phel|null|scalar`)?

Reply:
Sure, no problem.